### DISCUSSION TITLE
LUKSO Proposals website

### DISCUSSION THREAD

**Discussion by YamenMerhi** [lukso-network/LIPs#101] [Ideas]
Following what Ethereum does for the EIPs, it makes sense to think in the future to have a website like eips.ethereum.org where the specs are shown. 

We should not expect everyone who wants to read the specs to go to GitHub (even if they are technical, it's cool to have a website also) might be cleaner/easier and will show the specs better. Let's Discuss :rocket:

**Comment by potions1** [comment]
Profile Rentals : CONECPT

User A Requests Posting:

User A initiates a request to post content on User B's profile and includes a fee for this service.
Incoming Token Received:

User A sends a token representing the payment for the posting service to User B.
Token Validation:

User B validates the incoming post request to ensure it is genuine and has the appropriate to audience.
Review Content:

User B reviews the content that User A wants to post on their profile. This may include an advertisement or other types of content.
Decision by User B:

User B has the option to either accept or decline the request based on the content's appropriateness or other criteria.

If accepted, User B provides permission for the content to be posted on their profile.

If declined, the transaction is canceled, and User A's content is not posted.

Posting on Lukso Blockchain:

If User B accepts the request, the Lukso blockchain is utilized to execute the posting transaction securely.
Content on User B's Profile:

The content written by User A is now posted on User B's profile, visible to their audience.

Wrtiten by @kingzoot27


Social profile ownership
being able to sell your social account experiment in decentralized platform selling your account to other users the bigger the following the more it should be hypothetically this is an aspect to social-fi that is very similar to sales of NFT's

**Reply by potions1** [reply]
SPR's and SPO's for short
